sdk/dotnet/Pulumi/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+static Pulumi.Utilities.OutputUtilities.GetValueAsync<T>(Pulumi.Output<T> output) -> System.Threading.Tasks.Task<T>
```

Comment:
We should probably add this to the `Shipped` file, otherwise we may forget

Reply:
The tooling always adds to the `Unshipped` file. So I should just move it immediately to the `Shipped` file?

Same problem with [Pulumi.Automation/PublicAPI.Unshipped.txt](https://github.com/pulumi/pulumi/blob/2e25e28debe7b798f25036ade8e9b250c30c5828/sdk/dotnet/Pulumi.Automation/PublicAPI.Unshipped.txt) then as those changes have all shipped AFAIK.

Reply:
Fixed both `Pulumi` and `Pulumi.Automation` public apis.

Reply:
I think it slipped from our (my) radar, unfortunately. We should probably ever change the shipped file only - we ship often enough to not care about this distinction, IMO.